content/en/docs/concepts/storage/volumes.md

```diff
@@ -955,7 +955,7 @@ For more information, see the [vSphere volume](https://github.com/kubernetes/exa
 
 {{< feature-state for_k8s_version="v1.26" state="stable" >}}
 
-In Kubernetes {{< skew currentVersion >}}, all operations for the in-tree `vsphereVolume` type
+In Kubernetes {{< skew currentPatchVersion >}}, all operations for the in-tree `vsphereVolume` type
```

Comment:
The original is fine; this statement applies to all patch versions in a minor release.